recipes/stereo-pipeline/build.sh

```diff
@@ -0,0 +1,186 @@
+
+# Build dependencies first
+
+# Multiview
+cd $SRC_DIR/MultiView
+mkdir -p build && cd build
+cmake ..                                          \
+    -DCMAKE_BUILD_TYPE=Release                    \
+    -DMULTIVIEW_DEPS_DIR=${PREFIX}                \
+    -DCMAKE_VERBOSE_MAKEFILE=ON                   \
+    -DCMAKE_CXX_FLAGS="-O3 -std=c++11 -Wno-error" \
+    -DCMAKE_C_FLAGS='-O3 -Wno-error'              \
+    -DCMAKE_INSTALL_PREFIX=${PREFIX}              \
+    $deployFlag
+make -j${CPU_COUNT} install
+
+# Geoid
+cd $SRC_DIR/geoids
+# Build
+${FC} ${FFLAGS} -fPIC -O3 -c interp_2p5min.f
+${FC} ${LDFLAGS} ${LIB_FLAG} -o libegm2008${EXT} interp_2p5min.o
+# Install
+mkdir -p ${PREFIX}/lib
+/bin/cp -fv libegm2008.* ${PREFIX}/lib
+GEOID_DIR=${PREFIX}/share/geoids
+mkdir -p ${GEOID_DIR}
+/bin/cp -fv *tif *jp2 ${GEOID_DIR}
+
+# libnabo
+cd $SRC_DIR/libnabo
+mkdir -p build && cd build
+cmake                                          \
+  -DCMAKE_BUILD_TYPE=Release                   \
+  -DCMAKE_CXX_FLAGS='-O3 -std=c++11'           \
+  -DCMAKE_C_FLAGS='-O3'                        \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX}             \
+  -DEIGEN_INCLUDE_DIR=${PREFIX}/include/eigen3 \
+  -DBUILD_SHARED_LIBS=ON                       \
+  -DCMAKE_VERBOSE_MAKEFILE=ON                  \
+  ..
+make -j${CPU_COUNT} install
+
+# libpointmatcher
+cd $SRC_DIR/libpointmatcher
+mkdir -p build && cd build
+cmake                                          \
+  -DCMAKE_BUILD_TYPE=Release                   \
+  -DCMAKE_CXX_FLAGS="-O3 -std=c++17"           \
+  -DCMAKE_C_FLAGS='-O3'                        \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX}             \
+  -DCMAKE_VERBOSE_MAKEFILE=ON                  \
+  -DCMAKE_VERBOSE_MAKEFILE=ON                  \
+  -DBUILD_SHARED_LIBS=ON                       \
+  -DEIGEN_INCLUDE_DIR=${PREFIX}/include/eigen3 \
+  ..
+make -j${CPU_COUNT} install
+
+# fgr
+cd $SRC_DIR/FastGlobalRegistration
+FGR_SOURCE_DIR=$(pwd)/source
+mkdir -p build && cd build
+INC_FLAGS="-I${PREFIX}/include/eigen3 -I${PREFIX}/include -O3 -L${PREFIX}/lib -lflann_cpp -llz4 -O3 -std=c++11"
+cmake                                        \
+  -DCMAKE_BUILD_TYPE=Release                 \
+  -DCMAKE_CXX_FLAGS="${INC_FLAGS}"           \
+  -DCMAKE_INSTALL_PREFIX:PATH=${PREFIX}      \
+  -DCMAKE_VERBOSE_MAKEFILE=ON                \
+  -DFastGlobalRegistration_LINK_MODE=SHARED  \
+  ${FGR_SOURCE_DIR}
+make -j${CPU_COUNT}
+# Install
+FGR_INC_DIR=${PREFIX}/include/FastGlobalRegistration
+mkdir -p ${FGR_INC_DIR}
+/bin/cp -fv ${FGR_SOURCE_DIR}/FastGlobalRegistration/app.h ${FGR_INC_DIR}
+FGR_LIB_DIR=${PREFIX}/lib
+mkdir -p ${FGR_LIB_DIR}
+/bin/cp -fv FastGlobalRegistration/libFastGlobalRegistrationLib* ${FGR_LIB_DIR}
+
+# s2p
+cd $SRC_DIR/s2p
+baseDir=$(pwd)
+export CFLAGS="-I$PREFIX/include -O3 -DNDEBUG -march=native"
+export LDFLAGS="-L$PREFIX/lib"
+# Extension
+if [ "$(uname)" = "Darwin" ]; then
+    EXT='.dylib'
+else
+    EXT='.so'
+fi
```

Comment:
You are redefining these from above. Also, you don't need them. See my other comment.